governance/xc_admin/packages/xc_admin_frontend/utils/lamportsToSol.ts

```diff
@@ -0,0 +1,5 @@
+import { LAMPORTS_PER_SOL } from '@solana/web3.js'
+
+export function lamportsToSol(lamports: bigint): string {
+  return (lamports / BigInt(LAMPORTS_PER_SOL)).toString()
```

Comment:
Are we ok with rounding it? division of bigints lead to another bigint and loses all the decimals.